types/tools/block-tool.d.ts

```diff
@@ -16,6 +16,11 @@ export interface BlockTool extends BaseTool {
    */
   sanitize?: SanitizerConfig;
 
+  /**
+   * @param {boolean} readOnly - render HTML on readonly mode
+   */
+  render(readOnly?: boolean): HTMLElement;
```

Comment:
Maybe it would better to pass this flag through the constructor?

Reply:
yeah, definitely